zjit/src/hir.rs

```diff
@@ -4097,36 +4095,36 @@ mod opt_tests {
 
         assert_optimized_method_hir("rest", expect![[r#"
             fn rest:
-            bb0(v0:ArrayExact):
-              Return v0
+            bb0(v0:BasicObject, v1:BasicObject):
```

Comment:
we should still have ArrayExact

Reply:
fixed at https://github.com/ruby/ruby/pull/13529/commits/b5b4ad18847e561a9257c65996fc5445b20fdc70